x-pack/plugins/observability/public/pages/slo_details/constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const DEFAULT_DATE_FORMAT = 'MMM D, YYYY @ HH:mm:ss.SSS';
```

Comment:
I would just colocate this in the Page component. I don't think there will be a lot of constants going forward.

Reply:
You mean duplicate the constant in both overview and wide_chart components?

Reply:
you should use the date format from Kibana advance settings like that
```ts
 const { uiSettings } = useKibana().services;
 const dateFormat = uiSettings.get('dateFormat');
```
<img width="1747" alt="image" src="https://user-images.githubusercontent.com/189600/222561705-7c426adb-612f-4b9e-9fce-53a64d77445a.png">

Reply:
same for this settings
<img width="1418" alt="image" src="https://user-images.githubusercontent.com/189600/222562721-e0db0ad3-2d65-4a6e-8d6f-0fcc13216e84.png">